vars/setupCommonPipelineEnvironment.groovy

```diff
@@ -80,6 +80,8 @@ void call(Map parameters = [:]) {
             customDefaults: parameters.customDefaults,
             customDefaultsFromFiles: customDefaultsFiles ])
 
+        piperLoadGlobalExtensions script: script, customDefaults: parameters.customDefaults, customDefaultsFromFiles: customDefaultsFiles
```

Comment:
I think you may have to adopt the unit test?

Reply:
Tests are green